From 9d914641db3c8c37aec8c690d4aa22339fd13749 Mon Sep 17 00:00:00 2001
From: Phillip Lougher <phillip@squashfs.org.uk>
Date: Sun, 31 Aug 2014 23:56:22 +0100
Subject: [PATCH 087/120] actions: Regularise SYNTAX_ERROR() statements

Get rid of the redundant "Action mode: " prefix and make the
error messages match others.

Signed-off-by: Phillip Lougher <phillip@squashfs.org.uk>
---
 squashfs-tools/action.c | 12 +++++-------
 1 file changed, 5 insertions(+), 7 deletions(-)

diff --git a/squashfs-tools/action.c b/squashfs-tools/action.c
index a522044..1becf48 100644
--- a/squashfs-tools/action.c
+++ b/squashfs-tools/action.c
@@ -1261,8 +1261,7 @@ static int parse_sym_mode_arg(char *arg, struct mode_data **head,
 parse_operation:
 	/* trap a symbolic mode with just an ownership specification */
 	if(*arg == '\0') {
-		SYNTAX_ERROR("Action mode: Expected one of '+', '-' or '=', "
-			"got EOF\n");
+		SYNTAX_ERROR("Expected one of '+', '-' or '=', got EOF\n");
 		goto failed;
 	}
 
@@ -1281,8 +1280,8 @@ parse_operation:
 			op = ACTION_MODE_SET;
 			break;
 		default:
-			SYNTAX_ERROR("Action mode: Expected one of '+', '-' or "
-				"'=', got '%c'\n", *arg);
+			SYNTAX_ERROR("Expected one of '+', '-' or '=', got "
+				"'%c'\n", *arg);
 			goto failed;
 		}
 	
@@ -1322,9 +1321,8 @@ parse_operation:
 					mode &= mask;
 					goto perms_parsed;
 				default:
-					SYNTAX_ERROR("Action mode: "
-						"unrecognised permission "
-						"'%c'\n", *arg);
+					SYNTAX_ERROR("Unrecognised permission "
+								"'%c'\n", *arg);
 					goto failed;
 				}
 	
-- 
2.8.3

